api/model/notification.js

```diff
@@ -0,0 +1,11 @@
+const Model = require('./base_model');
+
+class Notification extends Model {
```

Comment:
Add documentation, what does this model represent?
What is the relationship with ArchiveNotification?